src/viam/examples/modules/complex/main.cpp

```diff
@@ -28,33 +28,32 @@
 using namespace viam::sdk;
 
 int main(int argc, char** argv) {
+    Registry::initialize();
     API base_api = API::get<Base>();
+    API gizmo_api = API::get<Gizmo>();
+    API summation_api = API::get<Summation>();
     Model mybase_model("viam", "base", "mybase");
 
+    // // CR erodkin: add note about how we've reduced the comment here.
+    // Make sure to explicity register resources with custom APIs.
```

Comment:
Removed the "must be done in `main` and not in resource implementation" bit of this comment because the idea of doing registration pre-main doesn't really make sense anymore.